roles/edpm_frr/tasks/download_cache.yml

```diff
@@ -3,6 +3,7 @@
 - name: Download needed container images
   containers.podman.podman_image:
     name: "{{ item }}"
+    auth_file: "{{ edpm_podman_auth_file | default() }}"
```

Comment:
per earlier comment ... then I think we could drop these

Reply:
In this case I do agree, as it keeps with the established pattern for vars of `download_cache.yml` tasks.